File: 3685-count-subarrays-of-length-three-with-a-condition/count-subarrays-of-length-three-with-a-condition.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var countSubarrays = function(nums) {
    let count = 0;
    for(i=0; i+2<nums.length;i++){
        if(nums[i] + nums[i+2] === nums[i+1]/2){
            count += 1;
        } else {
            continue;
        }
    }
    return count;
};

File: 3685-count-subarrays-of-length-three-with-a-condition/count-subarrays-of-length-three-with-a-condition.py
class Solution:
    def countSubarrays(self, nums: List[int]) -> int:
        count = 0
        i = 0
        while i + 2 < len(nums):
            if nums[i] + nums[i+2] == nums[i+1]/2:
                count += 1
                i += 1
            else:
                i += 1
        return count        